src/flashhls-adapter.js

```diff
@@ -333,8 +352,55 @@ class FlashHLSAdapter extends FakeEventTarget {
     this.ended = false;
     this.seeking = false;
     this.duration = null;
+    this.currentTime = null;
     this.buffer = null;
     this.watched = null;
+    this._startTime = null;
+    this._firstPlay = true;
+  }
+
+  /**
+   * attach media - return the media source to handle the video tag
+   * @public
+   * @returns {void}
+   */
+  attachMediaSource(): void {
+    this.attach();
+    this._startTimeAttach = this._lastTimeDetach;
```

Comment:
clean `this._startTimeAttach` on destroy (change media)

Reply:
done